src/test/java/uk/gov/companieshouse/api/strikeoffobjections/interceptor/authorization/AuthorizationIntegrationTest.java

```diff
@@ -1,4 +1,4 @@
-package uk.gov.companieshouse.api.strikeoffobjections.authorization;
+package uk.gov.companieshouse.api.strikeoffobjections.interceptor.authorization;
```

Comment:
This integration test was specifically targeting the original interceptor I wrote. I think extending it my adding more tests to check all the new interceptors work in the context of the Spring Boot app - and together - would be very useful...